integration/gitopscreateprofile_test.go

```diff
@@ -8,21 +8,25 @@ import (
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+
 	"github.com/spf13/afero"
+
+	. "github.com/weaveworks/eksctl/integration/runner"
 )
 
 var _ = Describe("(Integration) generate profile", func() {
 
 	Describe("when generating a profile", func() {
 		It("should write the processed repo files in the supplied directory", func() {
 
-			eksctlSuccess("generate", "profile",
+			cmd := eksctlExperimentalCmd.WithArgs(
```

Comment:
Need to add this above:
```
			if clusterName = "" {
				clusterName = "amazing-testing-gopher"
			}
			if region = "" {
				region = "us-west-2"
			}
```

Reply:
could we initialize the `clusterName` at the beginning of the integration test for all the suites at once?

Reply:
could we initialize the clusterName at the beginning of the integration test for all the suites at once?

Reply:
Yes, good idea. I just thought it maybe better to do that in a separate PR.

Reply:
Yes, please, I had troubles with hardcoded-values mutating the state of the tests as they run, like the above ones, when running tests on an existing cluster, see also: https://github.com/weaveworks/eksctl/pull/1148/commits/df6773f4045218ccb4bf32321d8fc87b6bc61048.